packages/lb-components/src/components/pointCloud2DRectOperationView/index.tsx

```diff
@@ -94,14 +95,14 @@ const PointCloud2DRectOperationView = (props: IPointCloud2DRectOperationViewProp
     return allRects;
   }, [pointCloudBoxList]);
 
-  const updateRectList = () => {
+  const updateRectList = useMemoizedFn(() => {
     const rectListByBoxList = getRectListByBoxList();
```

Comment:
getRectListByBoxList is not static method or class' fn, state change will not get new state value